tests/Api/Responses/shop/checkout/cart/update_cart.json

```diff
@@ -6,28 +6,26 @@
     "shippingAddress": {
         "@id": "\/api\/v2\/shop\/addresses\/@integer@",
         "@type": "Address",
-        "firstName": "Jane",
+        "firstName": "Oliver",
```

Comment:
It's just an idea. What if you marked fields that should be updated with a prefix, for example `Updated: `. It would be easier to ensure that the expected fields were actually updated without checking each field in the sent request one by one in the test.